lib/g_searcher_web/views/report_view.ex

```diff
@@ -1,3 +1,13 @@
 defmodule GSearcherWeb.ReportView do
   use GSearcherWeb, :view
+
+  @completed "Completed"
+  @searching "Searching"
+
+  def status(%{html_cache: html}) when is_binary(html),
```

Comment:
Maybe when `not is_nil(html)` since we don't really care much about the data type 🤔